R/connection-pane.R

```diff
@@ -404,15 +416,46 @@ get_feature_table_columns <- function(table, host, token) {
 }
 
 get_clusters <- function(host, token) {
+  #browser()
   clusters <- db_cluster_list(host = host, token = token)
   purrr::map_dfr(clusters, function(x) {
+
     list(
       name = as.character(glue::glue("[{x$state}] {x$cluster_name} ({x$cluster_id})")),
       type = "cluster"
     )
   })
 }
 
+
+get_clusters_2 <- function(host, token) {
```

Comment:
This function loads clusters into the brixter.env and just shows two leaves underneath. Those leaves are expanded on consecutive click when clusters are filtered from the brixter.env$cluster